### PR TITLE
manifest: Update west manifest to add LVGL shell support

### PR DESCRIPTION
**Pull request by faxe1008**
Updates the west manifest to add an LVGL shell with initial memory statistics and monkey testing support.

Relates to [lvgl/pull/42](https://github.com/zephyrproject-rtos/lvgl/pull/42)

### CODE REVIEW

**Comment by fabiobaltieri** [comment]
@faxe1008 should this PR also have a `CONFIG_LV_Z_SHELL=y` in the LVGL sample so the new code gets built in CI?

**Comment by faxe1008** [comment]
@fabiobaltieri Good catch :^).  would you have it enabled by default or added only  for e.g. in `native_posix` board conf?

**Comment by fabiobaltieri** [comment]
> @fabiobaltieri Good catch :^). would you have it enabled by default or added only for e.g. in `native_posix` board conf?

I think it would make sense to enable it in `samples/subsys/display/lvgl/prj.conf`, so all platform when building that sample.

File: samples/subsys/display/lvgl/prj.conf [hunk 1]
```diff
@@ -12,4 +13,5 @@ CONFIG_LV_USE_LOG=y
 CONFIG_LV_USE_LABEL=y
 CONFIG_LV_USE_BTN=y
 CONFIG_LV_USE_IMG=y
+CONFIG_LV_USE_MONKEY=y
```
Comment (jfischer-no) [inline samples/subsys/display/lvgl/prj.conf:16]: How does it affect flash memory usage?

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: samples/subsys/display/lvgl/prj.conf (hunk 1 above)
Comment (faxe1008) [reply samples/subsys/display/lvgl/prj.conf:17]: The `monkey` is an opaque structure and is allocated within the memory pool the zephyr manages for lvgl.
So the only overhead is actual code size for that feature (and the shell commands).

Built the sample for `mimxrt1060_evk`.
Without `CONFIG_LV_USE_MONKEY`:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      255856 B         8 MB      3.05%
             RAM:     1077224 B        32 MB      3.21%
            ITCM:          0 GB       128 KB      0.00%
            DTCM:          0 GB       128 KB      0.00%
           OCRAM:          0 GB       768 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
```

With `CONFIG_LV_USE_MONKEY=y`:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      257720 B         8 MB      3.07%
             RAM:     1077224 B        32 MB      3.21%
            ITCM:          0 GB       128 KB      0.00%
            DTCM:          0 GB       128 KB      0.00%
           OCRAM:          0 GB       768 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
```
`257720-255856=1864`, so roughly `1KiB`.

**Review by jfischer-no — DISMISSED** [review]

**Review by gmarull — DISMISSED** [review]

File: west.yml [hunk 2]
```diff
@@ -260,7 +260,7 @@ manifest:
       revision: ce57712f3e426bbbb13acaec97b45369f716f43a
       path: modules/lib/loramac-node
     - name: lvgl
-      revision: f7cf01d413aa9e76f376e25aa7b5c08dfdbc3c24
+      revision: pull/42/head
```
Comment (fabiobaltieri) [inline west.yml:263]: 5da257f782a8f9c6e265bdc60ebc2a93fdee24de all yours

Comment (faxe1008) [reply west.yml]: Update the revision :^)

**Review by fabiobaltieri — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Pull request merged**